app/controllers/repairs_controller.rb

```diff
@@ -4,34 +4,38 @@
 class RepairsController < ApplicationController
   before_action :authenticate_user!
   def index
-    @repairs = Repair.all
+    @repair = Repair.all
   end
 
   def new
-    @repairs = Repair.new
+    @repair = Repair.new
   end
 
   def create
-    @repairs = Repair.new(repair_params)
-    @repairs.save
-    redirect_to repairs_path
+    @repair = Repair.new(repair_params)
+    if current_user.nil?
```

Comment:
Not quite correct here.  current_user will not be nil.  However, current_user.owner may be nil, which is the problem we are trying to catch.